Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -2235,20 +2221,54 @@ void IMixedRealitySourceStateHandler.OnSourceLost(SourceStateEventData eventData
             }
         }
 
-        void IMixedRealityPointerHandler.OnPointerDragged(MixedRealityPointerEventData eventData) { }
+        #endregion IMixedRealitySourceStateHandler implementation
+
+        #region IMixedRealityInputHandler implementation
 
+        /// <inheritdoc/>
         void IMixedRealityInputHandler.OnInputUp(InputEventData eventData)
         {
-            if (shouldSwallowEvents)
+            if (isDragging)
             {
-                // Prevents the handled event from PointerUp to continue propagating
                 eventData.Use();
-                shouldSwallowEvents = false;
             }
         }
 
-        void IMixedRealityInputHandler.OnInputDown(InputEventData eventData) { }
+        void IMixedRealityInputHandler.OnInputDown(InputEventData eventData)
+        {
+            if(isDragging)
```

Comment:
When does this OnInputDown handler occur? I'm curious why the scrollingobjectcollection needs to handle both input down AND pointer events, seems like it should only need to handle one. Perhaps this was a mistake / could be improved decision in the initial experimental PR?